shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -142,14 +168,9 @@ public static void main(String[] argv) throws IOException {
 
     // Validate command args
     if (args.length < 2) {
-      System.out.format("Command %s requires at least %s arguments (%s provided)%n",
-              2, argv.length);
-      shell.printUsage();
+      printHelp(String.format("Command %s requires at least %s arguments (%s provided)%n",
```

Comment:
There are 4 formatters in this string but I only see 2

Reply:
Nice catch! Remove the extra one.